GrailDiary/GrailDiary/View Controllers/AddPOIViewController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  AddPOIViewController.swift
+//  GrailDiary
+//
+//  Created by Ian Becker on 5/14/20.
+//  Copyright © 2020 Ian Becker. All rights reserved.
+//
+
+import UIKit
+
+protocol AddPOIDelegate {
+    func poiWasAdded(_ poi: POI)
+}
+
+class AddPOIViewController: UIViewController {
+    
+    @IBOutlet weak var locationVC: UITextField!
+    @IBOutlet weak var countryVC: UITextField!
+    @IBOutlet weak var clue1VC: UITextField!
+    @IBOutlet weak var clue2VC: UITextField!
+    @IBOutlet weak var clue3VC: UITextField!
+    
+    var delegate: AddPOIDelegate?
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+    }
+    
+    @IBAction func save(_ sender: Any) {
+        
+        guard let location = locationVC.text, let country = countryVC.text else { return }
+        
+        var clues: [String] = []
+        
+        if let clue1 = clue1VC.text, clue1 != "" {
```

Comment:
A "swifty" way to check is a string is empty is !clue1.isEmpty. Good tip for future projects.

Reply:
I knew there had to be a better way! Thanks!